cosmos-init/plugins/packages/cosmosc2-tool-common/src/components/TargetPacketItemChooser.vue

```diff
@@ -233,7 +233,7 @@ export default {
         return
       }
       this.internalDisabled = true
-      const cmd = this.mode === 'tlm' ? 'get_all_telemetry' : 'get_all_commands'
+      const cmd = this.mode === 'tlm' ? 'get_all_telemetry_list' : 'get_all_commands_list'
       this.api[cmd](this.selectedTargetName).then((packets) => {
```

Comment:
All we ever use is the packet_name and description so this saves a MASSIVE amount of data round tripping